superdesk/io/feeding_services/twitter.py

```diff
@@ -43,11 +43,11 @@ class TwitterFeedingService(FeedingService):
         },
         {
             'id': 'access_token_key', 'type': 'text', 'label': 'Twitter Access Token Key',
-            'placeholder': 'Twitter access_token_key', 'required': True
+            'placeholder': 'Twitter access_token_key', 'required': False
         },
         {
             'id': 'access_token_secret', 'type': 'password', 'label': 'Twitter Access Token Secret',
-            'placeholder': 'Twitter access_token_secret', 'required': True
+            'placeholder': 'Twitter access_token_secret', 'required': False
```

Comment:
No longer use, make it optional in configuration instead of removing it to avoid confusing. Should we remove it?

Reply:
If it's not used anymore it's better rather remove it to avoid confusion.
I assume this update should not break already running twitter ingests, right?

Reply:
Yes, There is **NO** breaking changes, we have run it for a few days and the old twitter ingests are working fine